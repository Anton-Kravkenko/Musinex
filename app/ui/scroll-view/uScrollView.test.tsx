import UScrollView from '@/ui/scroll-view/uScrollView'
import Title from '@/ui/title/title'
import { render } from '@testing-library/react-native'

describe('uScrollView', () => {
	test('Render', () => {
		const uScrollViewRender = render(
			<UScrollView
				style={{
					backgroundColor: 'red'
				}}
				paddingBottom={100}
				className='mb-5'
				pointerEvents={'box-none'}
				contentContainerStyle={{
					backgroundColor: 'blue'
				}}>
				<Title
					numberOfLines={2}
					style={{ backgroundColor: 'red' }}
					size={25}
					className='mb-5'>
					test
				</Title>
			</UScrollView>
		)
		expect(uScrollViewRender).toMatchSnapshot()
	})
})
