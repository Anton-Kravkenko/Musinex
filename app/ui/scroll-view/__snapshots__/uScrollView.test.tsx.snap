// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uScrollView Render 1`] = `
<RCTScrollView
  className="mb-5"
  contentContainerStyle={
    {
      "backgroundColor": "blue",
    }
  }
  decelerationRate="fast"
  pointerEvents="box-none"
  renderToHardwareTextureAndroid={true}
  showsHorizontalScrollIndicator={false}
  showsVerticalScrollIndicator={false}
  style={
    {
      "backgroundColor": "red",
    }
  }
>
  <View>
    <View
      style={
        [
          {
            "backgroundColor": "#1B2021",
            "borderRadius": 6,
            "height": 50,
            "width": 40,
          },
          {
            "backgroundColor": "red",
          },
        ]
      }
      testID="skeleton"
    />
  </View>
</RCTScrollView>
`;
